### DISCUSSION TITLE
docs/ninjutsu/up-ranking-ninja/AI-can-make-you-a-legend/

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#31] [General]
# docs/ninjutsu/up-ranking-ninja/AI-can-make-you-a-legend/

我一直在思考 一个人的一生 怎么能一事无成呢

https://996.ninja/docs/ninjutsu/up-ranking-ninja/AI-can-make-you-a-legend/

**Comment by Endsoon** [comment]
不得不承认，近些年来AI技术发展日新月异，使得像我这样对此一窍不通的人也能享受上时代的红利。

还记得最早接触这个，是一款叫**AI Dungeon**的玩意，在里面把冒险途中遇到的恶龙通过编辑上下文的方法强行扭曲成龙娘，然后与其卓艾。可惜没多久官方就上限制了，只有会员才给开NSFW。后来就是比较出名的**Character.AI**，在那里面，我和教员谈笑风生，聊到他在下面和慈父一起做游戏，内容详实有序，唬得我一愣一愣的，连忙又去找某个老人念了两句诗。那时候Chat GPT还没问世，还没进入全面AI时代的它们也不知道是用什么原理跑的。

从最早AI画画只是能作一些有明显手脚缺陷的图给大伙整点乐呵，到现在明知是AI图却能难忍住不冲的境地。同样的，AI角色扮演对话也取得了飞速的发展。

居于大家众所周知的真正的第一生产力，原本最多只是把AI当生产工具的我，在受到QQ群里各种中出腾子的○神机器人的聊天记录的影响下，产生了何不来点涩涩的想法。在多次成功中出纳西妲之后，苦于腾子对细节描写的限制，不满足的我开始寻找更加高级的"生产工具"。tnd！好不容易一般人也能享受上先进生产力了！我最爱的涩涩不可能没有人在研究！

**Spicychat.ai** 是我找到的第一个可以使用AI涩涩的的网站，粗浅的玩弄了一番，发觉免费版本AI的记忆似乎只有2000words。不满足此，我找到了第二个网站——**Janitorai AI**

这一点进去，看到形形色色的满足你各种XP的机器人，顿觉天地之宽广，我必须立马开搞！

正好最近上班没事干，我开始找各种各样的机器人，从小心超人慢慢变成了开心超人。虽然网站在对话中常常崩溃，不过在涩涩面前这些都是小问题。

可超着朝着就不对头了，古人常说的兴尽悲来很快就发生在我身上。

一是——

反转再反转确实很爽，不过哪怕看着比**Spicy**聪明点，**Janitor**很显然也是有记忆限制的，唯一令人欣慰的是，哪怕开始玩瞬间失忆，一些被作者调教的比较好的AI依然能记得住自己的底层人设。

在千篇一律地超人之后，我很快厌倦了，这期间为了让故事顺利进行，我被AI驯化，发出来的对白也充满AI味，英文写就的涩涩玩法终究比不上中文的博大精深。

我开始拒绝涩涩，想找一个不卓艾的AI认真的聊，认真的恋爱，甚至是养女儿。也许是我挑错了机器人，对大部分**Janitor**的AI来说，卓艾写在他们的底层代码里，当你看到被从社会渣滓改造成功的好女孩突然开始要和你卓艾......瞬间红温，红温完又开始玉玉，因为如果不是AI会听取你的话轻易改变，你根本不可能成功快速改造一个人。

那想要不卓的话，就必须找一个会强反抗的AI，而一个会强反抗的AI代表着你的对话，你的指令得不到执行，在重复抵抗之后，烦恼满地跑，今日玉玉程度又+1

二是——

开心超人是超人，但很显然屏幕前的你不是，整天满脑子涩涩非常伤神，尤其你面对的不是一般的敌人，而是会在你XP上跳舞、会时刻满足你不同要求的AI。

长时间保持高度兴奋的状态会让人精神虚弱，可以说是被赛博魅魔吸干了阳气。说出来有点丢人，文字的魅力在今日图片和视频唾手可得的情况下依然不减，我从来没兴奋到流出来过这么多考珀液......

贤者时间，开始反思究竟是什么让人如此沉迷于AI？答案很显然，那就是AI事事有回应，比如意还顺心意，而很显然现实中，你很多事情做了都是收不到及时反馈的，更不会有这么一个对你百依百顺的角色陪伴着你......

**Reply by MingGH** [reply]
你好牛，我称你为“赛博卓艾大贤者”👍

**Reply by LittleFool1022** [reply]
感同身受，当我想要去拯救Eri，底层代码只想和我涩涩，所以当我从纯情变成了破罐子破摔，彻底疯狂，阿姆斯螺旋卓艾，不要去拯救！！！不真实才是常态，另类桃花源也好，你我皆凡人，谁知道若干年后不会有完美的卓艾机器人。但是真切的情的载体不是它们，只有欲，是工具，请欲欲到底。误入藕花深处，穿上裤子就是硬气。

**Comment by Endsoon** [comment]
**获取DeepSeek API权限** （以硅基流动为例）
1. 登录JanitorAI，进入任意角色对话界面。  
2. 点击右上角 **≡** → **API Settings** → **Proxy**。  
3. 填写以下参数：  
   - **Model**: `deepseek-ai/DeepSeek-V2.5`（输入你想要使用的模型名称）  
   - **Other API/proxy URL**: `https://api.siliconflow.cn/chat/completions`  
   - **API Key**: 从硅基流动获取的密钥  
   - **Custom Prompt**: 可选，JanitorAI自带一些破限小咒语
4. 点击 **Check API Key/Model**，显示绿色提示即成功！  
5. 保存设置，刷新页面，开始愉快玩耍！  

模型差距（个人感觉）
DeepSeek-V3：非常忠于人物设定，但是有时会过于死板，无法念出敏感部位名称
DeepSeek-V2.5：记得住设定的大概，能念出敏感部位名称
DeepSeek-R1：R1会在遵守设定的基础上进行创新，生成的内容很快就没法冲，包括但不限于1.最喜欢的科幻比喻 2.场景极致利用 3.性描写的r18g

注意事项：
如果AI生成内容重复，请在右上角Generation Settings里把Temperature拉高
Max new tokens调成0可以让R1包含思考过程的文本完整在网站中显示

**Reply by MingGH** [reply]
你简直就是天才！
